src/js/providers/html5.js

```diff
@@ -569,7 +571,7 @@ define([
         };
 
         this.preload = function(item) {
-            let preload = item.sources[_currentQuality] ? item.sources[_currentQuality].preload : 'metadata';
+            const preload = item.sources[_currentQuality] ? item.sources[_currentQuality].preload : 'metadata';
             _setAttribute('preload', preload);
```

Comment:
In the near-term we should insure that the source cannot be changed here. The "preload" is a player property. It will not longer be an item or source property. To ensure this preload should only accept preload values "auto", "metadata" and "none" ("none" would only be needed to stop preloading which is not something we need yet). Providers will be single item use - initialized or constructed with an item which cannot change (preload and load will not need to accept an item) https://github.com/jwplayer/jwplayer/issues/2130 .